docs/source/use-cases.rst

```diff
@@ -0,0 +1,375 @@
+   B = [model_b(x, reference) for x in data]
+
+   roll_A = [roll(A[i], A[i + 1]) for i in range(len(A) - 1)]
+   roll_B = [roll(B[i], B[i + 1]) for i in range(len(B) - 1)]
+   compare = [compare_ab(a, b) for a, b in zip(A, B)]
+
+   results = summarize(compare, roll_A, roll_B)
+
+These models are time consuming and need to be run on a variety of inputs and
+situations.  The analyst has his code now as a collection of Python functions
+and is trying to figure out how to parallelize such a codebase.  They use
+dask.delayed to wrap their function calls and capture the implicit parallelism.
+
+.. code-block:: python
+
+   from dask import compute, delayed
+
+   data = [delayed(load)(fn) for fn in filenames]
+   reference = delayed(load_from_database)(query)
+
+   A = [delayed(model_a)(x, reference) for x in data]
+   B = [delayed(model_b)(x, reference) for x in data]
+
+   roll_A = [delayed(roll)(A[i], A[i + 1]) for i in range(len(A) - 1)]
+   roll_B = [delayed(roll)(B[i], B[i + 1]) for i in range(len(B) - 1)]
+   compare = [delayed(compare_ab)(a, b) for a, b in zip(A, B)]
+
+   lazy_results = delayed(summarize)(compare, roll_A, roll_B)
+
+They they depend on the dask schedulers to run this complex web of computations
+in parallel.
+
+.. code-block:: python
+
+    results = compute(lazy_results)
+
+They appreciate how easy it was to transition from the experimental code to a
+scalable parallel version.  This code is also easy enough for their teammates
+to understand easily and extend in the future.
+
+
+Algorithm developer
+~~~~~~~~~~~~~~~~~~~
+
+A graduate student in machine learning is prototyping novel parallel
+algorithms.  They are in a situation much like the financial analyst above
+except that they need to benchmark and profile their computation heavily under
+a variety of situations and scales.  The dask profiling tools (:doc:`single
+machine<diagnostics>` and `distributed machine`_) provide the feedback they
+need to understand their parallel performance, including how long each task
+takes, how intense communication is, and their scheduling overhead.  They scale
+their algorithm between 1 and 50 cores on single workstations and then scale
+out to a cluster running their computation at thousands of cores.  They don't
+have access to an institutional cluster, so instead they use dask-ec2_ to
+easily provision clusters of varying sizes.
+
+Their algorithm is written the same in all cases, drastically reducing the
+cognitive load, and letting the readers of their work experiment with their
+system on their own machines, aiding reproducibility.
+
+.. _`distributed machine`: http://distributed.readthedocs.io/en/latest/web.html
+.. _dask-ec2: http://distributed.readthedocs.io/en/latest/ec2.html
+
+
+Scikit-Learn or Joblib User
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A data scientist wants to scale their machine learning pipeline to run on their
+cluster to accelerate parameter searches.  They already use ``sklearn``'s
```

Comment:
If I remember right, sphinx doesn't like it when `` is followed by anything other than a space, so this might not be formatted properly (I might be wrong here though).